sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -70,7 +70,7 @@ been renamed to `KeyCurveName`, `KeyOperation`, and `KeyType`, respectively.
 - `Key` now has attribute `properties`, which holds certain properties of the
 key, such as `version`. This changes the shape of the returned `Key` type,
 as certain properties of `Key` (such as `version`) have to be accessed
-through the `properties` property. See the updated [docs](https://azure.github.io/azure-sdk-for-python/ref/azure.keyvault.keys.html)
+through the `properties` property.
```

Comment:
Sorry to keep picking on these Key Vault files but  now we have an orphaned "for details." line after each of these because PR suggestions can't include unchanged lines 😞

Reply:
removed them